api/src/main/java/io/minio/messages/NotificationConfiguration.java

```diff
@@ -42,42 +42,55 @@ public class NotificationConfiguration {
   @ElementList(name = "TopicConfiguration", inline = true, required = false)
   private List<TopicConfiguration> topicConfigurationList;
 
+  @Element(name = "EventBridgeConfiguration", required = false)
+  private EventBridgeConfiguration eventBridgeConfiguration;
+
   public NotificationConfiguration() {}
 
   /** Returns cloud function configuration. */
   public List<CloudFunctionConfiguration> cloudFunctionConfigurationList() {
-    return Collections.unmodifiableList(
-        cloudFunctionConfigurationList == null
-            ? new LinkedList<>()
-            : cloudFunctionConfigurationList);
+    return Utils.unmodifiableList(cloudFunctionConfigurationList);
   }
 
   /** Sets cloud function configuration list. */
   public void setCloudFunctionConfigurationList(
       List<CloudFunctionConfiguration> cloudFunctionConfigurationList) {
     this.cloudFunctionConfigurationList =
-        Collections.unmodifiableList(cloudFunctionConfigurationList);
+        cloudFunctionConfigurationList == null
+            ? null
+            : Utils.unmodifiableList(cloudFunctionConfigurationList);
   }
 
   /** Returns queue configuration list. */
   public List<QueueConfiguration> queueConfigurationList() {
-    return Collections.unmodifiableList(
-        queueConfigurationList == null ? new LinkedList<>() : queueConfigurationList);
+    return Utils.unmodifiableList(queueConfigurationList);
   }
 
   /** Sets queue configuration list. */
   public void setQueueConfigurationList(List<QueueConfiguration> queueConfigurationList) {
-    this.queueConfigurationList = Collections.unmodifiableList(queueConfigurationList);
+    this.queueConfigurationList =
+        queueConfigurationList == null ? null : Utils.unmodifiableList(queueConfigurationList);
   }
 
   /** Returns topic configuration list. */
   public List<TopicConfiguration> topicConfigurationList() {
-    return Collections.unmodifiableList(
-        topicConfigurationList == null ? new LinkedList<>() : topicConfigurationList);
+    return Utils.unmodifiableList(topicConfigurationList);
   }
 
   /** Sets topic configuration list. */
   public void setTopicConfigurationList(List<TopicConfiguration> topicConfigurationList) {
-    this.topicConfigurationList = Collections.unmodifiableList(topicConfigurationList);
+    this.topicConfigurationList =
+        topicConfigurationList == null ? null : Utils.unmodifiableList(topicConfigurationList);
+  }
+
+  public EventBridgeConfiguration eventBridgeConfiguration() {
+    return this.eventBridgeConfiguration;
+  }
+
+  public void setEventBridgeConfiguration(EventBridgeConfiguration config) {
+    this.eventBridgeConfiguration = config;
   }
+
+  @Root(name = "EventBridgeConfiguration")
+  public static class EventBridgeConfiguration {}
```

Comment:
do we plan to implement this in future with details?

Reply:
I do not see any fields in S3 specification. It is empty ATM